### ISSUE TITLE
System.NotSupportedException: 'Unknown column type DateTime('UTC')'

### ISSUE THREAD

**Issue by dimenzzz** [killwort/ClickHouse-Net#95]
## Description
"System.NotSupportedException: 'Unknown column type DateTime('UTC')' " after update to version 1.2.1.

```
System.NotSupportedException
  HResult=0x80131515
  Message=Unknown column type DateTime('UTC')
  Source=ClickHouse.Ado
  StackTrace:
   at ClickHouse.Ado.Impl.ColumnTypes.ColumnType.Create(String name)
   at ClickHouse.Ado.Impl.Data.ColumnInfo.Read(ProtocolFormatter formatter, Int32 rows)
   at ClickHouse.Ado.Impl.Data.Block.Read(ProtocolFormatter formatter)
   at ClickHouse.Ado.Impl.ProtocolFormatter.ReadPacket(Response rv)
   at ClickHouse.Ado.Impl.ProtocolFormatter.ReadResponse()
   at ClickHouse.Ado.ClickHouseCommand.Execute(Boolean readResponse, ClickHouseConnection connection)
   at ClickHouse.Ado.ClickHouseCommand.ExecuteNonQuery()
   at ConsoleApp1.Program.Main(String[] args)
```
## Steps to reproduce
1.  CREATE TABLE test (`time` DateTime('UTC')) ENGINE = MergeTree() ORDER BY time
2. 
```C#
using var connection = new ClickHouseConnection(new ClickHouseConnectionSettings("Compress=False;CheckCompressedHash=False;Compressor=lz4;Host=localhost;Port=9000;Database=test;User=default;Password="));
connection.Open();

using var cmd = connection.CreateCommand();
cmd.CommandText = "select * from test";
cmd.ExecuteNonQuery();
```

**Comment by apalchevski** [comment]
Same problem when i try to write down DateTime? value to Nullable(DateTime('UTC')) column.

**Comment by AntonAlekseevAA** [comment]
Same problem, we fixed that using implicit cast to `Datetime` in query. Try to use
```
    SELECT col1, col2,  CAST(colWithDate AS Datetime)
    FROM test
```
This cheat cast you clear info about timezone in clickhouse server response and library can parse response correctly.

**Comment by killwort** [comment]
Released as 1.2.6.
